pkg/agent/endpoints/workload/handler.go

```diff
@@ -99,6 +100,7 @@ func (h *Handler) FetchJWTSVID(ctx context.Context, req *workload.JWTSVIDRequest
 		}
 
 		spiffeIDs = append(spiffeIDs, spiffeID)
+		hints[spiffeID.String()] = entry.Hint
```

Comment:
@guilhermocc Is there any scenario where it would make sense to have two entries with the same SPIFFE ID but different hints where it would also be meaningful to distinguish between the SVIDs corresponding to those entries? The only case I could imagine could be something like one of the entries has DNS names and the other doesn't, but I can't think of how it would be actually useful to use different hint values to differentiate the SVIDs in that case.

Reply:
85a408701d362616571f176e0b26d06d19fdd5d3